source/frontend/src/components/common/buttons/Button.tsx

```diff
@@ -66,7 +66,6 @@ const StyledButton = styled(BootstrapButton)`
     align-items: center;
     justify-content: center;
     padding: 0.4rem 1.2rem;
-    min-height: 3rem;
```

Comment:
ooph, this seems scary, are you sure that this doesn't have any unintended consequences?

Reply:
Pretty confident, was causing misalignment on a  lot of pages

Reply:
Its subtle, but this has definitely re-introduced the issue where the save and cancel buttons aren't the same size:

![image](https://github.com/user-attachments/assets/526b721b-74fb-47c8-b128-1e574ec4d02d)

Could you either revert this, or find a way of sizing the cancel button (with its additional border) the same as the save button?